package.json

```diff
@@ -20,6 +20,8 @@
         "deploy:goerli": "npx hardhat run --network goerli scripts/deploy.ts",
         "deploy:base_goerli": "npx hardhat run --network base_goerli scripts/deploy.ts",
         "deploy:xdai": "npx hardhat run --network xdai scripts/deploy.ts",
+        "deploy:xdai-bullaFinance": "npx hardhat run --network xdai scripts/deploy-bullaFinance.ts",
```

Comment:
not sure we need this again.

Reply:
should I just use it for deployment and then delete it?

Reply:
You can just run the script without adding it to package.json

Reply:
Ok - removed them in the latest update